infracost-usage-example.yml

```diff
@@ -51,6 +51,11 @@ resource_usage:
     monthly_handler_operations: 10000 # Monthly number of non-free handler operations (resources outside of the AWS::*, Alexa::*, and Custom::* namespaces).
     monthly_duration_secs: 0 # Monthly duration of non-free handler operations that go above 30 seconds, in seconds.  
 
+  aws_cloudtrail.my_cloudtrail:
+    monthly_additional_management_events: 1000 # Monthly additional copies of read and write management events. The first copy of management events per region is free, so this should only be non-zero if there are multiple trails recording management events in this region.
+    monthly_data_events: 2000 # Monthly data events delivered to S3, Lambda or DynamoDB
+    monthly_insight_events: 4000 # Monthly CloudTrail Insight events
```

Comment:
Not sure what's the best way to mention these numbers are `100k event amounts` e.g. 20 = 2000000 events

Reply:
👍